src/index.html

```diff
@@ -4,19 +4,105 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+      content="width=device-width, initial-scale=1.0"
     />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="/src/images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
```

Comment:
```suggestion
        <a href="#" class="logo">
          <img
            src="/src/images/logo.png"
            alt="logo"
          />
        </a>
```
looks like we can simplify it